src/website/apps/event/serializers.py

```
@@ -88,6 +236,24 @@ def get_event_image_url(self, obj):
             return obj.event_image.url  # Directly use the .url attribute
         return None
 
+    def get_organizers_count(self, obj):
+        try:
+            return obj.event_organizer_links.count()
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)